skills/ios-simulator-testing/scripts/log_monitor.py

```
@@ -0,0 +1,486 @@
+
+        # Setup signal handler for graceful interruption
+        def signal_handler(sig, frame):
+            self.interrupted = True
+            if self.log_process:
+                self.log_process.terminate()
+
+        signal.signal(signal.SIGINT, signal_handler)
+
+        try:
+            # Start log streaming process
+            self.log_process = subprocess.Popen(
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)